ajax/libs/jquery.i18n/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "jquery.i18n",
+  "description": "jQuery.i18n is a jQuery based Javascript internationalization library. It helps you to internationalize your web applications easily.",
+  "filename": "jquery.i18n.js",
```

Comment:
Let's modify `jquery.i18n.js` to `jquery.i18n.min.js`.
Because we should have a filename to point to the main file of a lib, if that file is not minified, please still use filename.min.js or filename.min.css structure naming, we'll do the minify job.
Sorry for not noticing this last time.
Ref: https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md#e-adding-a-new-library-by-a-single-packagejson